website/versioned_docs/version-5.x/ecosystem-angularjs.md

```diff
@@ -212,6 +212,79 @@ Migrating an existing AngularJS application to single-spa can be a tricky. Here
 console.log('legacyAngularjsApp status', singleSpa.getAppStatus('legacyAngularjsApp');
 ```
 
+### Step 2: Convert to SystemJS module:
+
+This step is not required unless you want to do [cross microfrontend imports](/docs/recommended-setup#cross-microfrontend-imports) between your angularjs microfrontend and other microfrontends.
+
+1. Add systemjs to your index.html file:
+
+```html
+<!-- consider checking/upgrading systemjs version -->
+<script type="systemjs-importmap">
+  {
+    "imports": {
+      "single-spa": "https://cdn.jsdelivr.net/npm/single-spa@5.9.1/lib/system/single-spa.min.js",
+      "@org/legacyAngularjsApp": "/main-angular-app.js"
+    }
+  }
+</script>
+<script src="https://cdn.jsdelivr.net/npm/systemjs@6.9.1/dist/system.min.js"></script>
+```
+
+2. Remove the global single-spa script:
+
+```diff
+- <script src="https://cdn.jsdelivr.net/npm/single-spa@5.9.1/lib/umd/single-spa.min.js"></script>
+```
+
+3. Modify your main / first angularjs script file to create a systemjs module instead of global variable. See [this code](/docs/ecosystem-angularjs#as-a-systemjs-module).
+
+4. Remove the `<script>` for loading that main / first angularjs script file. Replace it with a System.import.
+
+```diff
+- <script src="/main-angular-app.js"></script>
+```
+
+5. Modify the `<script>` in your main HTML file to load the angularjs app as a systemjs module instead of global variable:
+
+```diff
+- window.singleSpa.registerApplication({
+-   name: "legacyAngularjsApp",
+-   app: window.legacyAngularjsApp,
+-   app: function() { return System.import('@org/legacyAngularjsApp'); },
```

Comment:
This diff doesn't seem correct when rendered.

![Screen Shot 2021-03-02 at 12 05 53 PM](https://user-images.githubusercontent.com/4202993/109701179-f5b75e00-7b4f-11eb-8c2b-0e15891d9608.png)

Is this supposed to remove the registerApplication? If so, then there are duplicate `app: ` lines.

Reply:
Fixed